R/customVarInfo.R

```diff
@@ -284,11 +283,25 @@ defaultVarInfo <- list(
       ret
     }
   ),
+  # S4
+  list(
+    name = 'S4',
+    doesApply = isS4,
+    childVars = function(v) {
+      names <- slotNames(v)
+      values <- lapply(names, function(s) slot(v, s))
+      list(values = values, names = names)
+    },
+    hasChildren = TRUE,
+    shortType = 'S4',
+    longType = 'S4',
+    includeAttributes = FALSE
```

Comment:
Shouldn't we show the attributes that are not simultaneously a slot?
E.g. `class`/`package` or any other attributes added by the user

I think this would be possible to achieve by keeping `includeAttributes = FALSE` and adding a list of custom attributes that includes all normal attributes except for those that are slots.

Reply:
Yes, that's the problem the first time I create S4 var info here. I'll make it show attributes with slots excluded.